atomic_reactor/build.py

```diff
@@ -166,13 +168,20 @@ def df_path(self):
 
     def set_df_path(self, path):
         self._df_path = path
-        self.set_base_image(df_parser(path).baseimage)
+        dfp = df_parser(path)
+        base = dfp.baseimage
+        if base is None:
+            raise RuntimeError("no base image specified in Dockerfile")
+        self.set_base_image(base)
         logger.debug("base image specified in dockerfile = '%s'", self.base_image)
-        if not self.base_image.tag:
-            self.base_image.tag = 'latest'
+        self.parent_images.clear()
+        for image in dfp.parent_images:
+            self.parent_images[image] = None
```

Comment:
when is self.parent_images ever referenced? you set it here, but AFAIK never use it again.

Reply:
it's filled in later in pull_base_image to record what local tag or digest is used for those images -- used to update the dockerfile FROMs and recorded in the build metadata

Reply:
Okay. You might want to add a note about that in the commit message.

Reply:
Will do!